Language/Statements/For/Asynchronous_For_in/enclosing_loop_t01.dart

```diff
@@ -33,7 +32,6 @@ Future test1() async {
   Expect.listEquals(['a', 'a1', 'a2', '1', '11', '12', '2', '21', '22'], log);
```

Comment:
Await in line 28? Similar consideration for other occurrences of `Stream.close()` or `Stream.cancel()`. I can see a few `.close()`, but there may be more in folded text.

Reply:
Added `await` to `close()` in `execution_A05_t05.dart`. In this test `await` cannot be added because it leads to a deadlock (stream, controlled by `StreamController` is suspended in enclosing `await for` and `await .close()` won't let it to be resumed)

Reply:
OK, thanks!